deepmd/pt/utils/dataloader.py

```
@@ -50,8 +50,12 @@
 
 
 def setup_seed(seed) -> None:
-    torch.manual_seed(seed)
-    torch.cuda.manual_seed_all(seed)
+    if isinstance(seed, (list, tuple)):
+        mixed_seed = mix_entropy(seed)
+    else:
+        mixed_seed = seed
```

Ruff (0.8.2)

Use ternary operator `mixed_seed = mix_entropy(seed) if isinstance(seed, (list, tuple)) else seed` instead of `if`-`else`-block

Replace `if`-`else`-block with `mixed_seed = mix_entropy(seed) if isinstance(seed, (list, tuple)) else seed`

(SIM108)